bbb-fsesl-client/src/main/java/org/freeswitch/esl/client/inbound/Client.java

```diff
@@ -479,20 +479,43 @@ public void run() {
                                         //Member transfered to another conf...
                                         listener.conferenceEventTransfer(uniqueId, confName, confSize, event);
                                         return;
+                                    //API has changed between freeswitch 1.4 and 1.6
+                                    } else if (eventFunc.equals("conference_api_sub_transfer")) {
+                                        //Member transfered to another conf...
+                                        listener.conferenceEventTransfer(uniqueId, confName, confSize, event);
+                                        return;
                                     } else if (eventFunc.equals("conference_add_member")) {
                                     	System.out.println("##### Client conference_add_member");
                                         listener.conferenceEventJoin(uniqueId, confName, confSize, event);
                                         return;
```

Comment:
Couldn't you do something like
```
} else if (eventFunc.equals("conference_add_member") or eventFunc.equals("conference_member_add")) {
```
since the handler is exactly the same?
Doing so, your pull request would be much simpler, with just a few modified lines. :smile: